git_pile/git_pile.py

```diff
@@ -150,6 +164,20 @@ def revert(self, other):
     def destroy(self):
         git("config --remove-section pile", check=False, stderr=nul_f, stdout=nul_f)
 
+    @classmethod
+    def help(cls, prefix=""):
+        attr_prefix = "_Config__attr_doc_"
+        attr_docs = (k for k in cls.__dict__.keys() if k.startswith(attr_prefix + prefix))
+        if not attr_docs:
+            return ""
+
+        ret = ["", "", "configuration:"]
+        for doc in attr_docs:
```

Comment:
Nitpick: I would use a different name for the variable (maybe `name` or `attr_name`). While we know that `doc` really contains attribute names, its name may give the impression that it contains the actual documentation string.